<launch>
   
  <!-- Buffer Server -->
  <node pkg="tf2_ros" type="buffer_server" name="tf2_buffer_server_plugs" output="screen">
    <remap from="tf2_buffer_server" to="tf2_buffer_server_plugs" />
    <param name="buffer_size" value="120.0"/>
  </node>

  <node pkg="pr2_plugs_actions" type="detect_plug.py" name="detect_plug" output="screen"/>
  <test test-name="detect_plug_test" pkg="pr2_plugs_actions" type="detect_plug_test"/>

</launch>
